packages/web-twig/src/Resources/components/Stack/README.md

```diff
@@ -48,15 +48,35 @@ Without lexer:
 {% endembed %}
 ```
 
+## Custom Spacing
+
+You can use the `spacing` prop to apply custom spacing between items. The prop
+accepts either a spacing token (eg. `space-100`) or an object with breakpoint keys and spacing token values.
+
+```twig
+<Stack hasSpacing spacing="space-1200">
```

Comment:
🤷 anyway it could be another PR? I understand what you mean, but it would be a deprecation etc. I suggest we discuss it on our thursdays meeting and optionally implement it in next PR.

Reply:
I agree with @crishpeen however I also think that it should exist as an option for most used cases. I am personally using `hasSpacing` in many places. So the most used options should be default or the shortcut like`noSpacing` or `hasSpacing` should be available. All other cases can be done using `spacing="<token>"`.

Reply:
OK